docs/source/ttnn/ttnn/dependencies/tt_lib.rst

```diff
@@ -307,6 +307,8 @@ Tensor elementwise operations
 
 .. autofunction:: tt_lib.tensor.div_no_nan
 
+.. autofunction:: tt_lib.tensor.unary_div_no_nan
```

Comment:
the naming convention is not being followed

Reply:
Hi @cfjchu We have updated the code and inserted the screenshots of the results : [Link](https://github.com/tenstorrent/tt-metal/pull/8255#issue-2285610587)